dracut/30ignition/ignition-reboot.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition (reboot)
+DefaultDependencies=false
+Before=ignition-complete.target
+
+# Make sure ExecStart= runs before we switch root
+Before=initrd-switch-root.target
```

Comment:
Hmm, we shouldn't need this. `ignition-complete.target` is ordered before the main `initrd.target`, which is reached  well before `initrd-switch-root.target`.

Reply:
Dropping this line..